input/dpdk.cpp

```diff
@@ -292,7 +292,7 @@ InputPlugin::Result DpdkReader::get(PacketBlock& packets)
     m_stats.receivedPackets += receivedPackets;
     m_stats.receivedBytes += packets.bytes;
 
-    return Result::PARSED;
+    return opt.pblock->cnt ? Result::PARSED : Result::NOT_PARSED;
```

Comment:
The opt structure is not defined when WITH_FLEXPROBE is defined. This should be fixed.

Reply:
Fixed

Reply:
Perfect! Thanks!